src/main/java/de/retest/recheck/ui/descriptors/StringAttribute.java

```diff
@@ -79,7 +79,7 @@ private static String escape( final String value ) {
 		if ( value == null ) {
 			return value;
 		}
-		return value.replaceAll( "]]>", "]]&gt;" );
+		return value.replace( "]]>", "]]&gt;" );
```

Comment:
Was this change intended? This was a regular expression before, which in my opinion is false, since `]` should be escaped&mdash;thus this change may be justified. Is it possible to lookup the author of this change and talk for the reason of [`replace` vs `replaceAll`](https://stackoverflow.com/a/10827900)?